prog1.py

```diff
@@ -0,0 +1,28 @@
+import psutil
+import subprocess
+import re
+import time
+
+file_num = 3
+file_size = 4096
+#free_space = 
+
+
+# Detect local mounted disk
+local_disk_mounts = []
+partitions = psutil.disk_partitions(all=False)
+pattern = re.compile(".*/dev/[h,s]d[a-z].*")
```

Comment:
Given regex doesn't handle local FSes created on LVM / disk-mapper devices like this:
```
/dev/mapper/rsws14_vg-lv_images on /images type ext4 (rw,relatime,data=ordered)
/dev/mapper/rsws14_vg-lv_docker_xfs on /var/lib/docker-bk type xfs (rw,relatime,attr2,inode64,noquota)
```